src/frontend/src/lib/components/wizards/auth/AuthWizard.svelte

```diff
@@ -82,7 +82,7 @@
   const handleContinueCreatePasskey = async (): Promise<void | "cancelled"> => {
     isAuthenticating = true;
     try {
-      onSignUp(await authFlow.createPasskey());
+      onSignIn(await authFlow.createPasskey());
```

Comment:
Probably it came because we saw that the accounts were not being used with continue and we expected them, but the problem was that it was calling the wrong prop